xdsl/xdsl_opt_main.py

```diff
@@ -2,11 +2,12 @@
 import sys
 import os
 from io import StringIO
-from xdsl.frontend.passes.desymref import Desymrefy
+from xdsl.frontend.passes.desymref import DesymrefyPass
```

Comment:
Can you order these a bit? Maybe group all passes?

Reply:
I could group the import lines, but to do a single line, we would have to reorganize them in the filesystem AFAIU.